datumaro/plugins/tiling/untile.py

```diff
@@ -1,3 +0,0 @@
-# Copyright (C) 2023 Intel Corporation
-#
-# SPDX-License-Identifier: MIT
```

Comment:
why should we remove this?

Reply:
I removed this file and rename it as `merge_tile.py`.